packages/office-ui-fabric-react/src/components/FocusTrapZone/FocusTrapZone.tsx

```diff
@@ -154,51 +227,25 @@ export class FocusTrapZone extends BaseComponent<IFocusTrapZoneProps, {}> implem
     if (this.props.onFocusCapture) {
       this.props.onFocusCapture(ev);
     }
-    if (ev.target !== ev.currentTarget) {
+
+    if (ev.target !== ev.currentTarget && !this._isBumper(ev.target)) {
       // every time focus changes within the trap zone, remember the focused element so that
       // it can be restored if focus leaves the pane and returns via keystroke (i.e. via a call to this.focus(true))
       this._previouslyFocusedElementInTrapZone = ev.target as HTMLElement;
     }
   };
 
-  private _onKeyboardHandler = (ev: React.KeyboardEvent<HTMLDivElement>): void => {
-    if (this.props.onKeyDown) {
-      this.props.onKeyDown(ev);
-    }
-
-    // If the default has been prevented, do not process keyboard events.
-    if (ev.isDefaultPrevented()) {
-      return;
-    }
```

Comment:
@jspurlin I see this was added in #4486, which basically allows consumers to block FTZ focus behavior. This PR will break this behavior, but I'm not sure how or why consumers would do this. What are valid use cases for this?

Reply:
There might be cases when a consumer might want to do something special in specific cases. For example, if TAB normally navigates linearly across focusable elements, but in some case(s) the consumer wants to move focus to a different element.

Removing this will break Office Online

Reply:
FocusTrapZone shouldn't be modifying or affecting focus behavior within it. It does not control focus within its contents anymore, so there's no longer any FTZ behavior to "stop" here.

If consumers of FTZ want to modify focus behavior, they can still do that. Nothing in FTZ is preventing key or focus handlers from working.

Reply:
Yeah, after talking offline, getting on the same page, and since the divProps are still getting applied to the div _onKeyboardHandler isn't needed and things should continue to work